e2e/driver/puppeteer-launch.ts

```diff
@@ -1,22 +1,15 @@
 import * as puppeteer from 'puppeteer';
+import {getChromeFlags} from './chrome-startup-flags';
+
+const isHeadless = !process.env.PUPPETEER_HEADLESS;
 
 const defaultLaunchOpts = {
-  headless: !process.env.PUPPETEER_HEADLESS,
+  headless: isHeadless,
   slowMo: 10,
   defaultViewport: null,
   devtools: false,
-  ignoreDefaultArgs: ['--disable-extensions'],
-  args: [
-    '--no-sandbox',
-    '--disable-setuid-sandbox',
-    '--disable-dev-shm-usage',
-    '--disable-gpu',
-    '--disable-background-timer-throttling',
-    '--disable-backgrounding-occluded-windows',
-    '--disable-renderer-backgrounding',
-    '--disable-web-security',
-    '--disable-features=IsolateOrigins,site-per-process',
-  ]
+  ignoreDefaultArgs: true,
```

Comment:
I don't see code that's handling this argument, `ignoreDefaultArgs`. How is it able to continue working if the type was changed form a string array to a boolean?

Reply:
Because `ignoreDefaultArgs` takes param either is `boolean` or `Array<string>`.
```
ignoreDefaultArgs <boolean|Array<string>> If true, then do not use puppeteer.defaultArgs(). If an array is given, then filter out the given default arguments. Dangerous option; use with care. Defaults to false.
```